dice/templates/index.html

```diff
@@ -0,0 +1,133 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+    <meta name="generator" content="Hugo 0.79.0">
+    <title>Dice Gallery</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+
+
+    <style>
+        .dice-view {
+            display: grid;
+            grid-template-columns: repeat(2, 50%);
+            justify-items: center;
+            justify-content: center;
+        }
+
+        .dice-view .dice-img {
+            margin: 10px;
+            max-height: 110px;
+        }
+
+        .dice-view-big {
+            display: grid;
+            grid-template-columns: repeat(auto-fill, 260px);
+            justify-items: center;
+            justify-content: center;
+        }
+
+        .dice-view-big .dice-img {
+            margin: 10px;
+            height: 240px;
+        }
+
+        .dice-caption {
+            text-align: center;
+        }
+    </style>
+
+
+</head>
+<body>
+
+<main>
+
+    <section class="py-5 text-center container">
+        <div class="row py-lg-5">
+            <div class="col-lg-6 col-md-8 mx-auto">
+                <h1 class="fw-light">CS 61A Hog Dice Gallery</h1>
+                <p class="lead text-muted">
+                    Your true colors shine<br/>
+                    Through the rolls of your pig dice<br/>
+                    Just like a code rainbow
+                </p>
+                <p>
+                    <a href="https://cs61a.org/proj/hog_dice_gallery">Go To Spec</a>
```

Comment:
Not sure everyone will understand "Go To Spec". Do you mean something like "View challenge description"?